expected-failures/brittany.txt

```diff
@@ -0,0 +1,611 @@
+Found .cabal file brittany.cabal, but it did not mention data/Test483.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test484.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test485.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test486.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test487.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test488.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test489.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test49.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test490.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test491.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test492.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test493.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test494.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test495.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test496.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test497.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test498.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test499.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test5.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test50.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test500.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test501.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test502.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test503.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test504.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test505.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test506.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test507.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test508.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test509.hs
+Found .cabal file brittany.cabal, but it did not mention data/Test51.hs
+data/Test51.hs
+@@ -1,14 +1,11 @@
+  {-# LANGUAGE DatatypeContexts #-}
+- data
+-   ( LooooooooooooooooooooongConstraint a
+-   , LooooooooooooooooooooongConstraint b
+-   ) =>
+-   MyRecord a b
+-   = MyConstructor
+-     { foo1, foo2
+-         :: loooooooooooooooooooooooooooooooong
+-         -> loooooooooooooooooooooooooooooooong
+-     , bar  :: a
+-     , bazz :: b
+-     }
+
++ data MyRecord a b = MyConstructor
++   { foo1,
++     foo2 ::
++       loooooooooooooooooooooooooooooooong ->
++       loooooooooooooooooooooooooooooooong,
++     bar :: a,
++     bazz :: b
++   }
++
+
+  AST of input and AST of formatted code differ.
```

Comment:
This is due to us (knowningly, see #725) not supporting `DatatypeContexts`, but it would be trivial to add.

All other new `expected-failures` seem to be known comment-related idempotency bugs.